src/app-layer-ftp.c

```diff
@@ -663,7 +600,7 @@ static int FTPParseRequest(Flow *f, void *ftp_state,
                 if (state->current_line_len + 1 > state->port_line_size) {
                     /* Allocate an extra byte for a NULL terminator */
                     ptmp = FTPRealloc(state->port_line, state->port_line_size,
-                                      state->current_line_len + 1);
+                                      state->current_line_len );
                     if (ptmp == NULL) {
                         if (state->port_line) {
                             FTPFree(state->port_line, state->port_line_size);
```

Comment:
for some reason this came back:
```
  CC       app-layer-ftp.o
app-layer-ftp.c:606:29: warning: Use of memory after it is freed
                            FTPFree(state->port_line, state->port_line_size);
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 warning generated.
```

Reply:
I wasn't able to reproduce this warning with clang-6 nor clang-8. What version was used when the warning was generated?

Reply:
This is with:
```
$ CC=clang-8 scan-build-8 make
```
We don't run it in the CI